multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/sql/create_unlink_file_content_function.sql

```diff
@@ -0,0 +1,11 @@
+CREATE OR REPLACE FUNCTION UNLINK_FILE_CONTENT() RETURNS TRIGGER AS
```

Comment:
Since in both normal execution and in the event of an exception we return  NULL, wouldn't it be better if this is a procedure rather than a function?